hazelcast/src/main/java/com/hazelcast/util/VersionCheck.java

```diff
@@ -116,23 +120,32 @@ private void doCheck(Node hazelcastNode, String version, boolean isEnterprise) {
                 ? Long.toString(MemoryUnit.BYTES.toGigaBytes(totalNativeMemorySize)) : "0";
         //Calculate connected clients to the cluster.
         Map<ClientType, Integer> clusterClientStats = hazelcastNode.clientEngine.getConnectedClientStats();
+        RuntimeMXBean runtimeMxBean = ManagementFactory.getRuntimeMXBean();
+
+        Long clusterUpTime = hazelcastNode.getClusterService().getClusterClock().getClusterTime()
+                - hazelcastNode.getClusterService().getClusterClock().getClusterStartTime();
+        String clusterId = hazelcastNode.getConfig().getGroupConfig().getName()
```

Comment:
Is this a good cluster-id? I guess you want it to be unique (sort of), why don't you create a UUID for cluster?